Core/Debugger/WebSocket/MemorySubscriber.cpp

```diff
@@ -203,3 +299,37 @@ void WebSocketMemoryWriteU32(DebuggerRequest &req) {
 	JsonWriter &json = req.Respond();
 	json.writeUint("value", Memory::Read_U32(addr));
 }
+
+// Write bytes to memory (memory.write)
+//
+// Parameters:
+//  - address: unsigned integer address for the start of the memory range.
+//  - base64: data to write, encoded as base64 string
+//
+// Response (same event name) with no extra data.
+void WebSocketMemoryWrite(DebuggerRequest &req) {
+	auto memLock = Memory::Lock();
+	if (!currentDebugMIPS->isAlive() || !Memory::IsActive())
+		return req.Fail("CPU not started");
```

Comment:
return void value?   valid but unusal.  I'd separate req.Fail and return into two statements (but this does have an attraction... hm).

also this is the only single line `if` without braces in this function

Reply:
I just removed the other braces, I want to keep the validations clear and succinct, so they don't slow down reading the other code.  Which is also why I prefer the return void.

-[Unknown]

Reply:
My eyes react to it with some confusion, but I better get used to it I guess. It is quite neat indeed.